README.md

```diff
@@ -86,6 +87,12 @@ You can jump right into editing this file [here](https://github.com/ErichDonGubl
 * Designing low latency DSP algorithms suitable for embedded use (common filters, analysis functions)
 * Library for nonlinear dynamical or chaotic systems (solvers, numeric methods etc.)
 
+#### Personal information management
+
+* Contacts via [vCard](https://en.wikipedia.org/wiki/VCard) have been implemented using [`vobject`](https://crates.io/crates/vobject), but no "high-level interface" yet exists using it or an alternative. <!-- FIXME: What does this actually mean? What APIs are missing/expected? -->
```

Comment:
About the high-level API thing: the `Vcard` type, for example, provides functions for getting data from a `Vcard` object: `Vcard::email() -> Vec<Email>`. But the `Email` type is a simple wrapper around `String` - Functionalty for parsing the email into an actual email object (from the `email` crate for example) is not there yet. Same for other getters all over the place!

Same for the setter functionality: They take `String` as parameter right now rather than a proper type which represents the parameter properly. `vcard.with_email(String::from("notanemailadress"))` is perfectly fine on the typelevel. This has to change - we want proper semantics in the API!

Same story for the `icalendar` parsing/emitting.

Reply:
`exists yet` instead of `yet exists`?